photon-lib/src/main/java/org/photonvision/RobotPoseEstimator.java

```diff
@@ -178,41 +182,31 @@ public void setLastPose(Pose2d lastPose) {
      *     pipeline results used to create the estimate
      */
     public Optional<EstimatedRobotPose> update() {
```

Comment:
Nit: do we want this class to own a PhotonCamera or just consume PhotonPipelineResults? I kinda feel the latter may be better but idk

Reply:
That does make sense, but it also needs the Transform3D for the camera's position. Would you pass that each time?

Reply:
Oh I see. I guess not?